vscode/src/azure/workspaceActions.ts

```diff
@@ -442,6 +442,31 @@ export async function queryWorkspace(workspace: WorkspaceConnection) {
   return;
 }
 
+// Drop properties we don't care to track and clean up the types.
+function responseJobToJob(job: ResponseTypes.Job): Job {
+  return {
+    id: job.id,
+    name: job.name,
+    target: job.target,
+    status: job.status,
+    outputDataUri: job.outputDataUri,
+    count:
+      typeof job.inputParams.count === "string"
```

Comment:
what happens if `parseInt` fails?

Reply:
It returns `NaN`. Is that handled further down the line? I'm not sure. I'm also not sure what happens if the type is not a string nor a number. I'll go ahead and make the sanitization a bit stricter here.